services/realtimeclock.md

```diff
@@ -0,0 +1,33 @@
+# Real time clock
+
+    
+    identifier: 0x1a8b1a28
+    extends: _sensor
+
+## Registers
+
+    ro now @ reading {
+        year: u16
+        month: u8
+        day: u8
+        day_of_week: u8
+        hour: u8
+        min: u8
+        sec: u8
+    }
+
+Current time in 24h representation. Default streaming period is 1 second.
```

Comment:
specify ranges - is month 1-12 or 0-11? similar for day, day of week; also say hour is 0-24 (I assume), min and sec 0-59 (I don't think we'll do leap seconds would we?)

we also need to say if, once set, the leap years are automatically detected (I guess yes)

Reply:
I've added a few ranges on the register.

Reply:
let's do 0-60 on seconds to allow for leap seconds